persistence/commons/src/main/java/org/apache/causeway/persistence/commons/integration/repository/RepositoryServiceDefault.java

```diff
@@ -78,6 +83,7 @@ public class RepositoryServiceDefault
     @Getter(onMethod_ = {@Override})
     final MetaModelContext metaModelContext;
 
+    private ThreadLocal<Map<Class, Boolean>> threadLocal = ThreadLocal.withInitial(HashMap::new);
```

Comment:
give the `threadLocal` field a better name.

Reply:
Fixed